templates/security/OAuthClientCredentialsFlowClient.swift.hbs

```diff
@@ -53,4 +31,54 @@ public class OAuthClientCredentialsFlowClient: AbstractOAuthFlowClient {
             try await authenticate(scopes: scopes)
         }
     }
+    
+    private actor AuthenticationManager {
```

Comment:
Maybe we should remove "Manager" from this name? The name as it stands _sounds_ like it's more general than it actually is. Maybe `Authenticator` or `ClientCredentialsAuthenticator` or something like that...

Reply:
Authenticator is clean, it is a private class at the end of the day so ClientCredentialsAuthenticator is overkill